src/node_contextify.cc

```diff
@@ -58,6 +58,9 @@ class ContextifyContext {
  public:
   ContextifyContext(Environment* env, Local<Object> sandbox_obj) : env_(env) {
     Local<Context> v8_context = CreateV8Context(env, sandbox_obj);
+    if (v8_context.IsEmpty())
```

Comment:
The `context_.IsEmpty()` check below already takes care of this.  `persistent_handle.Reset(isolate, empty_handle)` is functionally equivalent to `persistent_handle.Reset()`.